### ISSUE TITLE
[Training Issue]  detection_eval 非常低

### ISSUE THREAD

**Issue by AnmachenGuo** [eric612/MobileNet-YOLO#194]
Model : yolov2 , yolov3 , ssd or segmentaion

Backbone : mobilenet , ... , mobilenet_yolov3_lite

Questions :

detection_eval 非常低，只有0.0*的级别，但是训练时输出log，recall看起来蛮正常的，所以找不到问题出在哪里，请教一下

**Comment by AnmachenGuo** [comment]
I1027 15:23:21.975855 53357 solver.cpp:253] Iteration 24480 (0.368664 iter/s, 27.125s/10 iters), loss = 1.5828
I1027 15:23:21.975891 53357 solver.cpp:272]     Train net output #0: det_loss1 = 0.6027 (* 1 = 0.6027 loss)
I1027 15:23:21.975899 53357 solver.cpp:272]     Train net output #1: det_loss2 = 0.807415 (* 1 = 0.807415 loss)
I1027 15:23:21.975905 53357 sgd_solver.cpp:121] Iteration 24480, lr = 0.000147856
I1027 15:23:49.939527 53357 solver.cpp:253] Iteration 24490 (0.357603 iter/s, 27.9639s/10 iters), loss = 1.71546
I1027 15:23:49.939671 53357 solver.cpp:272]     Train net output #0: det_loss1 = 0.633981 (* 1 = 0.633981 loss)
I1027 15:23:49.939680 53357 solver.cpp:272]     Train net output #1: det_loss2 = 1.32662 (* 1 = 1.32662 loss)
I1027 15:23:49.939687 53357 sgd_solver.cpp:121] Iteration 24490, lr = 0.000147856
I1027 15:24:04.143505 53357 yolov3_layer.cpp:532] noobj: 0.00880216 obj: 0.326824 iou: 0.756567 cat: 1 recall: 0.977503 recall75: 0.547569 count: 42
I1027 15:24:04.185156 53357 yolov3_layer.cpp:532] noobj: 0.00112894 obj: 0.0917843 iou: 0.518392 cat: 1 recall: 0.611513 recall75: 0.0738586 count: 31
I1027 15:24:19.438052 53357 solver.cpp:443] Iteration 24500, Testing net (#0)
I1027 15:24:19.438099 53357 net.cpp:679] Ignoring source layer label_data_1_split
I1027 15:24:19.438555 53357 net.cpp:679] Ignoring source layer Yolov3Loss1
I1027 15:24:19.438562 53357 net.cpp:679] Ignoring source layer Yolov3Loss2
I1027 15:24:36.345458 53357 solver.cpp:550] class1: 0.0943547
I1027 15:24:36.345633 53357 solver.cpp:556]     Test net output #0: detection_eval = 0.0943547

**Comment by lqian** [comment]
noobj: 0.00112894 obj: 0.0917843，说明训练不正常。如果没有修改骨干网，建议使用迁移学习，能比较快速的获得高精度模型

**Comment by Serissa** [comment]
你好！我用迁移学习训练，也出现了同样的问题，Train net output：det_loss1， det_loss2的损失迭代100次后小于1.1e-8，我是按照yolov3中的solver.prototxt来修改训练的，请问哪里出了问题？

**Comment by AnmachenGuo** [comment]
已解决，吧test.prototxt的batchsize改成1就好了，之前设的8

**Comment by AnmachenGuo** [comment]
已解决，把test.prototxt的batchsize改成1就好了，之前设的8
从这里得到的启发 http://www.caffecn.cn/?/question/892
感觉很玄学
